.github/workflows/ort.yml

```diff
@@ -0,0 +1,189 @@
+
+name: The OSS Review Toolkit (ORT)
+
+on:
+    schedule:
+      - cron: "0 0 * * *"
+    push:
+        tags:
+            - "ort-for-v*.*"
+jobs:
+    run-ort:
+        name: Create attribution files
+        runs-on: ubuntu-latest
+        strategy:
+          fail-fast: false
+        env: 
+          PYTHON_ATTRIBUIOTNS: "python/THIRD_PARTY_LICENSES_PYTHON"
```

Comment:
I'm not sure what `ATTRIBUIOTNS` is?  I don't suppose it's supposed to be `ATTRIBUTIONS` or `ATTRIBUTES`?

Reply:
Third party attribution files

Reply:
```suggestion
          PYTHON_ATTRIBUTIONS: "python/THIRD_PARTY_LICENSES_PYTHON"
```

Reply:
oops. thanks!